docs/changelog.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+* :feature:`869` Added some customized menu items to the applciation menu. Users can now access the `Usage Guide`, `FAQ`, `Issues & Feature Requests`, and `Logs Directory` from within the Help menu. Additionally, there is a `Get Premium` menu item for quick access to the rotki.com premium site. Finally, both backend and frontend logs (``rotkehlchen.log`` and ``rotki-electron.log`` respectively) are now found in these standard locations per OS:
+
+  * Linux: ``~/config/rotki/logs``
+  * OSX: ``~/Library/Application Support/rotki/logs``
+  * Windows: ``<WindowsDrive>:\Users\<User>\Roaming\rotki\logs\``
```

Comment:
Can you also change the log locations per OS in `installation_guide.rst`?

Reply:
done https://github.com/rotki/rotki/pull/869/commits/dd6612f5b6440fd4a27372ffc0b34bfe9fe34e88